AirCasting/Utils/AppInformations.swift

```diff
@@ -0,0 +1,15 @@
+// Created by Lunar on 12/07/2021.
+//
+
+import Foundation
+import SwiftUI
+
+extension UIApplication {
```

Comment:
as this is an extension the convention is to name files in format: `WhatIsBeingExtended+Description.swift`. In this case it might be `UIApplication+AppInfo.swift`

Reply:
Didn't know, thank You! 😄